admin/config/solidus_admin/tailwind.config.js.erb

```diff
@@ -82,5 +83,7 @@ module.exports = {
     require('@tailwindcss/aspect-ratio'),
     require('@tailwindcss/typography'),
     require('@tailwindcss/container-queries'),
+    plugin(({ addVariant }) => addVariant('hidden', '&([hidden])')),
```

Comment:
Am I right that we are not using these modifiers anywhere?

Reply:
Thoughts about moving this to that other PR? If, for some reason, the reviews of that one will change what we are anticipating, we might end up not using these modifiers and forget about them.